apps/web/app/(booking-page-wrapper)/team/[slug]/[type]/queries.ts

```diff
@@ -132,7 +133,10 @@ export async function getCRMData(
   let crmAppSlug = Array.isArray(crmAppSlugParam) ? crmAppSlugParam[0] : crmAppSlugParam;
   let crmRecordId = Array.isArray(crmRecordIdParam) ? crmRecordIdParam[0] : crmRecordIdParam;
 
-  if (!teamMemberEmail || !crmOwnerRecordType || !crmAppSlug) {
+  const shouldSkipCrmRefetch =
+    crmFetchAttempted === "true" || (Array.isArray(crmFetchAttempted) && crmFetchAttempted[0] === "true");
```

Comment:
Why do we consider crmFetchAttempted as an array ? We should assume it to be a boolean consider "true" to be only truthy value and other values to be false.